marekventur/dependency-updater
# Update "sinon" to version 1.17.3

<pre>1.17.3 / 2016-01-27
===================

  * Add spy.notCalled to documentation
    Fix issue [#969](https://github.com/cjohansen/Sinon.JS/issues/969) and add spy.notCalled to docs
  * Remove `sinon.test()` and `sinon.testCase`.
    Functionality has been extracted out into the `sinon-test` module allowing us to remove these two tightly coupled methods from sinon's public API.
  * Merge pull request [#972](https://github.com/cjohansen/Sinon.JS/issues/972) from jonnyreeves/feature/remove-global-logger
    Remove `sinon.log` and `sinon.logError`
  * Remove `sinon.log` and `sinon.logError`
    These two functions introduced global, mutable state to the sinon instance; sinon.log has been removed and sinon.logError has been replaced by `sinon.configureLogError` which will return a `logError` function using the supplied options.  Modified FakeXMLHttpRequest and FakeXDR constructors to pass configuration down to the instances they create and fakeServer to allow configuration to be supplied.
  * Merge pull request [#923](https://github.com/cjohansen/Sinon.JS/issues/923) from ichthala/better_restore_docs
    Clarify documentation on creating stubs and spies
  * Merge pull request [#967](https://github.com/cjohansen/Sinon.JS/issues/967) from jonnyreeves/feature/decouple-format
    Remove the use of `sinon.format` from the codebase
  * Merge pull request [#968](https://github.com/cjohansen/Sinon.JS/issues/968) from jonnyreeves/feature/organise-tests
    Re-organise tests (removes `sinon-test.js`)
  * De-fluff
  * Remove `sinon-test` module.
    Kills off a single test for `sinon.log` - I'm not convinced anything of value was lost :)
  * Extract `get-config` tests from `sinon-test`.
  * Extract `function-to-string` tests from `sinon-test`.
    Pulls out a suite into `test/util/core/funciton-to-string-test` for better organisation.
  * Extract `restore` tests from `sinon-test`.
    Pulls out a suite into `test/util/core/restore-test` for better organisation.
    As a side note, I am not sure that `sinon.restore` is a core util; should it infact be part of the public API and be moved to `lib/`?
  * Extract `createStubInstance` tests from `sinon-test`
    Pulls out a suite into `test/stub-test` to align with the fact `createStubInstance` is defined (and exposed against) the `spy` module.
  * Extract `deep-equal` tests from `sinon-test`.
    Pulls out a suite into `test/util/core/deep-equal-test` for better organisation.
  * Extract `wrap-method` tests from `sinon-test`.
    Pulls out 3 suites into `test/util/core/wrap-method-test` for better organisation.
  * Extract `extend` tests from `sinon-test` to `extend-test`
  * Move 'lib/util/core' tests into 'test/util/core'
  * Remove the use of `sinon.format` from the codebase
    The `test-call` suite now check that the `util/core/format` module was invoked by asserting against the returned string value; this allows us to import `util/core/format` inside both `spy.js` and `call.js` instead of using `sinon.format`
  * Merge pull request [#936](https://github.com/cjohansen/Sinon.JS/issues/936) from jonnyreeves/feature/cjsify-sandbox
    Extract `sandbox` into a CommonJS module.
  * Merge pull request [#964](https://github.com/cjohansen/Sinon.JS/issues/964) from raejin/master
    Fix stub.withArgs unable to recognize sinon matchers
  * Require sinon.deepEqual in a more modular way
    per @mantoni suggestion, thanks!
  * Fix 648: test for this.proxy before trying toString on it
  * use the correct sinon.deepEqual to test sinon matcher
  * add stub test to ensure sinon matcher is recognized within stub.withArgs
  * Merge pull request [#963](https://github.com/cjohansen/Sinon.JS/issues/963) from kellyselden/patch-1
    update repo link
  * update repo link
  * Merge pull request [#962](https://github.com/cjohansen/Sinon.JS/issues/962) from mroderick/update-dependencies
    Update dependencies
  * Remove unused dependency util
    Thank you bithound!
  * Update samsam
  * Update lolex
  * Update browserify
  * Update dependency pre-commit</pre>
